pkg/proxy/endpoints_test.go

```diff
@@ -495,35 +495,35 @@ func TestUpdateEndpointsMap(t *testing.T) {
 	}
 
 	testCases := []struct {
-		// previousEndpoints and currentEndpoints are used to call appropriate
+		// previousEndpointSlices and currentEndpointSlices are used to call appropriate
 		// handlers OnEndpointSlice* (based on whether corresponding values are nil
 		// or non-nil) and must be of equal length.
 		name                           string
-		previousEndpoints              []*discovery.EndpointSlice
-		currentEndpoints               []*discovery.EndpointSlice
-		oldEndpoints                   map[ServicePortName][]*BaseEndpointInfo
+		previousEndpointSlices         []*discovery.EndpointSlice
+		currentEndpointSlices          []*discovery.EndpointSlice
+		oldEndpointsMap                map[ServicePortName][]*BaseEndpointInfo
```

Comment:
If `previousEndpointSlices` and `oldEndpointsMap` are just different representations of the same data (which I think they are?) then that would be more obvious if they both used the same adjective ("previous" or "old").

Reply:
renamed `oldEndpointsMap` -> ` previousEndpointsMap`